src/css/nav.css

```diff
@@ -299,3 +299,12 @@ ul.nav-list > li:first-child[data-depth="6"] {
 .nav-panel-explore .component .title {
   font-family: var(--body-font-family-bold);
 }
+
+.nav-link.beta::after {
```

Comment:
If it's a cloud beta, would it be better to use the `Cloud beta` image? Maybe make it smaller so it looks nicer

Reply:
Also use a `cloud-beta` class so we have the option to display the normal `Beta` image for non-cloud docs that are beta?